doc/Development_Documentation/05_Objects/01_Object_Classes/05_Class_Settings/25_Inheritance.md

```diff
@@ -49,6 +49,10 @@ the attributes directly, you will not get the inherited values.
 > **Bear in mind**
 > The complex data type field collections does not support inheritance.
 
+Getting values from an object without inheritance
+````php
+\Pimcore\Model\DataObject\AbstractObject::setGetInheritedValues(false); // default = true
+``
```

Comment:
```suggestion
```php
$inheritanceModeBackup = DataObject::getGetInheritedValues();
\Pimcore\Model\DataObject\AbstractObject::setGetInheritedValues(false); // default = true

// ... your code goes here
// ...
// ...

// set status back to the original value 
DataObject::setGetInheritedValues($inheritanceModeBackup);

```

Please don't forget to reset 

```